.github/CODEOWNERS

```diff
@@ -47,6 +50,7 @@ api-report/azure-client.api.md                                            @micro
 api-report/azure-service-utils.api.md                                     @microsoft/fluid-cr-api
 api-report/test-client-utils.api.md                                       @microsoft/fluid-cr-api
 api-report/fluid-framework.api.md                                         @microsoft/fluid-cr-api
+api-report/fluid-static.api.md                                            @microsoft/fluid-cr-api @microsoft/fluid-cr-framework
```

Comment:
Will revert this change. Got fluid-framework and fluid-static mixed up in my head.